### DISCUSSION TITLE
Disable certain locks in August account

### DISCUSSION THREAD

**Discussion by Hulkkih** [homebridge-plugins/homebridge-august#37] [General]
Is it possible to not implement all locks in August account? I would like to use only one lock. These give me errors and simply hiding them in accecories wont help.

**Comment by donavanbecker** [comment]
Why can't you hide them with the plugin config?

**Reply by Hulkkih** [reply]
Will the hide function disable that lock loaded by plugin? These two locks will give me error all the time because they are not added two Yale bridge so simply hiding them is not enough.

**Reply by donavanbecker** [reply]
> Will the hide function disable that lock loaded by plugin? These two locks will give me error all the time because they are not added two Yale bridge so simply hiding them is not enough.

Have you tried? It should.

**Reply by Hulkkih** [reply]
Yea, its working now, thanks. Hided them adding commands to JSON config. It also now shows in plugin settings UI. Think I missed that earlier. 

Another problem though.. Lock stays that "opening" state. Is it known problem? Any solution?

**Reply by donavanbecker** [reply]
I will have to open and issue up upstream. If you have an issue though please open an issue and not a discussion, so it's easier to try with details I need to best troubleshoot.
